drivers/amf/stable/tools/tests/StreamingSDKTests/jobs/Scripts/run_programs/run_streaming.py

```diff
@@ -617,3 +618,50 @@ def __find_by_xpath(self, xpath, driver, wait=5):
             return element
         except TimeoutException:
             raise ValueError(f"Could not find element with XPATH: {xpath}")
+
+
+class RunParsec(RunStreaming):
+
+    def __init__(self, args, case: dict[str, Any], *,
+                 socket, script_path: Optional[str]) -> None:
+        super().__init__(args, case)
+        if not script_path:
+            raise ValueError("Script path is required to launch Parsec")
+        if not socket:
+            raise ValueError("Socket is required to launch Parsec")
+        self.socket = socket
+        self.script_path = script_path
+
+    def run_on_win(self):
+
+        self.__delete_log()
+
+        if self.args.execution_type != "server":
```

Comment:
Так условие чуть более явно
```suggestion
        if self.args.execution_type == "client":
```

Reply:
Это условие в том числе для нескольких клиентов, если такое нам не нужно, то я поправлю